client/my-sites/stats/feedback/modal/index.tsx

```diff
@@ -90,11 +117,22 @@ const FeedbackModal: React.FC< ModalProps > = ( { siteId, onClose } ) => {
 					onChange={ setContent }
 				/>
 				<div className="stats-feedback-modal__button">
+					{ ! isCheckingAbilityToSubmitFeedback && ! isAbleToSubmitFeedback && (
```

Comment:
I think it makes sense to move this check above the `TextareaControl` and completely hide the text area and button if submissions are currently disabled. If someone isn't paying attention, they might write their feedback before noticing they can't submit it.

Reply:
A note on disabled buttons. I've seen it happen a few times were someone sees a disabled button and doesn't understand why they can't click on it. So while it seems obvious to us, I think it would be even better to just hide the submission-related elements when blocked.

Reply:
Addressed in [Disable textarea](https://github.com/Automattic/wp-calypso/pull/94197/commits/b50f3e6748bad6332b63c88bbce6977ba5bc7f48).